mozregression/main.py

```diff
@@ -79,6 +79,9 @@ def parse_args(argv=None):
                         help=("last known good revision (use with"
                               " --inbound)."))
 
+    parser.add_argument("--hunt-fix", action="store_true",
```

Comment:
General comment: I think I slightly prefer the nomenclature of "find fix" as opposed to "hunt fix", just because the former ("find") is more commonly used for these sorts of things. Could you do a global s/hunt/find/ on this? :)